docs/source/pdbqt_spec.rst

```diff
@@ -0,0 +1,79 @@
+
+PDBQT Format for Coordinate Files
+=================================
+
+This is copied straight from the Autodock 4.2 manual. 
+It might be helpful to have a record of it here. 
+
+**Extension**: .pdbqt
+
+.. code-block::
+
+    “ATOM◼◼%5d◼%-4s%1s%-3s◼%1s%4d%1s◼◼◼%8.3f%8.3f%8.3f%6.2f%6.2f%4s%6.3f◼%2s \n",
+    atom_serial_num, atom_name, alt_loc, res_name, chain_id, res_num, ins_code, x, y, z, occupancy, temp_factor, footnote, partial_charge, atom_type
+
+(The “◼” symbol is used to indicate one space.)
+
+The PDBQT format adds four things to standard formatted PDB files:
+
+1. partial charges are included in each ATOM or HETATM record, substituting for the fields in columns 67-76.
+
+2. AutoDock atom types (which may be one or two letters) are included in each ATOM or HETATM record in columns 78-79.
```

Comment:
atom types do take more up to three characters for ADGPU and Vina, i'd write from column 78 until next whitespace or similar